Clarify feed is and Subject (#176)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Feed:

> : A collection of Signed Statements, observed by the Transparency Service, related to the `sub` field of the 'CWT_Claims' property of the protected header of Signed Statements.

Reviewer: ```suggestion
: An unordered collection of arbitrary data (artifacts, signed statements, receipts, or other data) related to the text identifiers, observed by the transparency service in the `sub` field of the CWT Claim property of the protected header of Signed Statements.
```

Author: Unordered: 
This isn't necessarily so. A transparency service may order them. They may even order by different parameters, which is implementation specific. Stating they're not ordered is an over-statement.
This looks like a lot of extra text, to demonstrate we really need to solve what gets persisted, and what gets returned.

Author: And, this same text has to be fixed whether it's on the feed or the subject property.

> Verifiers use the Feed to ensure completeness and Non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the Artifact they are evaluating.

Reviewer: ```suggestion
```

Author: A feed returns the collection of signed statements (or transparent statements). It returns what's on the append-only log. 
It simply returns what the transparency service persisted.

Author: See https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/176/commits/e674b8c1c9b19dbbbe8fc75dd4ac993670438e3d for clarification 

> Feed:
>
> : A collection of Receipts, as recorded by the Transparency Service, based on filtering of properties from the envelope including, but not limited to the `sub` field of the `CWT_Claims`.

Subject:

> : This term has the same definition as in RFC8392.

Reviewer: ```suggestion
: This term has the same definition as in RFC8392, which relies on the definition in RFC7519, which is repeated here with some modifications for readability: The "sub" (subject) claim identifies the principal that is the subject of the CWT.  The claims in a CWT are normally statements about the subject.  The subject value MUST either be scoped to be locally unique in the context of the issuer or be globally unique. The processing of this claim is generally application specific.  The "sub" value is a case-sensitive string containing a StringOrURI value.```

Reviewer: It seems essential to repeat this definition based on the discussions we have had regarding identifiers and their duration.

Author: done
